packages/components/src/circular-option-picker/types.ts

```diff
@@ -40,22 +41,57 @@ export type CircularOptionPickerProps = {
 	 * The child elements.
 	 */
 	children?: ReactNode;
+};
+
+type WithBaseId = {
+	baseId: string;
+};
+
+type FullListboxCircularOptionPickerProps = CommonCircularOptionPickerProps & {
 	/**
-	 * Whether the keyboard interaction should wrap around.
+	 * Whether the control should present as a set of buttons,
+	 * each with its own tab stop.
+	 *
+	 * @default false
+	 */
```

Comment:
The extra default value is unnecessary and causes Storybook to display it twice in its generated docs.

```suggestion
	 * Whether the control should present as a set of buttons,
	 * each with its own tab stop.
	 */
```